packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -202,20 +205,31 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   }
 
   private _getStickyPlaceholderHeight(isSticky: boolean): React.CSSProperties {
-    const height = this.nonStickyContent ? this.nonStickyContent.offsetHeight : 0;
-
+    let height = 0,
```

Comment:
Can simplify this to 2 lines:

`const height = this.nonStickyContent ?  this.nonStickyContent.offsetHeight : 0;`
`const width = this.nonStickyContent ? this.nonStickyContent.scrollWidth : 0;`

Reply:
@leddie24 
const height = this.nonStickyContent ? this.nonStickyContent.offsetHeight : 0;
const width = this.nonStickyContent ? this.nonStickyContent.scrollWidth : 0; 
here this.nonStickyContent null check 'll be done twice